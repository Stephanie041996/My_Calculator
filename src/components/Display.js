import React from 'react';
import PropTypes from 'prop-types';

function Display({ Calculate }) {
  return (
    <>
      <input id="result" value={Calculate || '0'} />
    </>
  );
}

Display.propTypes = {
  Calculate: PropTypes.string.isRequired,
};
export default Display;
