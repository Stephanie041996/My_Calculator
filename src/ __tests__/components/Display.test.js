import React from 'react';
import renderer from 'react-test-renderer';
import Display from '../../components/Display';

describe('Display', () => {
  it('should display result', () => {
    const tree = renderer.create(<Display />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
