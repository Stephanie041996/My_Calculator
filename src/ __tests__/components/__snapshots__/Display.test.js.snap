// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display should display result 1`] = `
<div
  className="display-result"
>
  <input
    defaultValue="0"
    id="result"
  />
</div>
`;
